### ISSUE TITLE
The performance on Object365

### ISSUE THREAD

**Issue by chensnathan** [IDEA-Research/DINO#109]
Hi,

Have the pre-trained DINO-SwinL been evaluated on the Object365 validation set? 
It would be much appreciated if it is possible to share the performance on the object365 dataset.

**Comment by FengLi-ust** [comment]
The performance of O365 val depends on the val images you choose. In our val set, the performance is around 45-47 AP.
We will consider releasing the checkpoint of this large model after this paper gets peer-reviewed.

**Comment by SlongLiu** [comment]
We use the first 5000 images in O365 as our val set, as mentioned in our paper.

**Comment by chensnathan** [comment]
Got it. Thanks for sharing the details and results. It helps a lot.
